main.py

```diff
@@ -2,6 +2,42 @@
 
 from django.db.models import QuerySet
 
+from db.models import Actor
+from db.models import Genre
+
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+
+    actors = [
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson")
+    ]
+
+    for genre in genres:
+        Genre.objects.create(
+            name=genre
+        )
+
+    for name, surname in actors:
+        Actor.objects.create(
+            first_name=name,
+            last_name=surname
+        )
+
+    Genre.objects.filter(name="Dramma").update(name="Drama")
+    Actor.objects.filter(
+        first_name="George", last_name="Klooney").update(
+        last_name="Clooney")
+    Actor.objects.filter(
+        first_name="Kianu", last_name="Reaves").update(
+        first_name="Keanu", last_name="Reeves")
```

Comment:
fix the breaking lines here: move first_name to the open quote or move closing quote to the next line